### DISCUSSION TITLE
I find it hard to find how to run a beta release

### DISCUSSION THREAD

**Discussion by KennethLavrsen** [blakeblackshear/frigate#5590] [General]
I am not strong on Docker. I follow a recipe and then it works

I am missing a recipe for how to write a docker_compose file which will pull in the beta

The beta documentation show a line

`image: ghcr.io/blakeblackshear/frigate:stable`

and that fails. You really have to be a hard core developer geek to know how to run a beta

**Comment by NickM-27** [comment]
You should be using the images at the bottom of the beta release notes. The stable tag won't have an image until 0.12 is released. The docs just exist there so they're correct once the beta is fully released.

**Comment by KennethLavrsen** [comment]
So it should for beta 8 say

`image: ghcr.io/blakeblackshear/frigate:0.12.0-beta8`

I could not find that earlier because in reality there are ways to read two release notes.

The one I looked at is the one you see when you click releases on this URL and the latest is listed on top of longer list

https://github.com/blakeblackshear/frigate/releases  And it lists source files at the bottom but you do not see image links

but if you click the headline of the release and see only that release on URL 

https://github.com/blakeblackshear/frigate/releases/tag/v0.12.0-beta8

Then the two links show up.  That is an odd thing in Github.

and there are two links. I assume the tensorrt is only for those testing the Nvidia stuff. I run on MacMini (intel)/Ubuntu with a Coral TPU

Thanks for the quick answer
